### PR TITLE
Fix "lexer" being used when uploading to bpaste.net

### LINKED ISSUE #5806: The --pastebin feature submits to bpaste.net with "python3" lexer instead of "text"
The `--pastebin` option currently submits the output of `pytest` to `bpaste.net` using `lexer=python3`: https://github.com/pytest-dev/pytest/blob/d47b9d04d4cf824150caef46c9c888779c1b3f58/src/_pytest/pastebin.py#L68-L73

For some `contents`, this will raise a "HTTP Error 400: Bad Request".

As an example:
~~~
>>> from urllib.request import urlopen
>>> with open("data.txt", "rb") as in_fh:
...     data = in_fh.read()
>>> url = "https://bpaste.net"
>>> urlopen(url, data=data)
HTTPError: Bad Request
~~~
with the attached [data.txt](https://github.com/pytest-dev/pytest/files/3561212/data.txt).

This is the underlying cause for the problems mentioned in #5764.

The call goes through fine if `lexer` is changed from `python3` to `text`. This would seem like the right thing to do in any case: the console output of a `pytest` run that is being uploaded is not Python code, but arbitrary text.

### PR DESCRIPTION
**Pull request by goerz**
Closes #5806 

Since this is a followup to #5764, I'm basing this on the `features` branch instead of `master`.

### CODE REVIEW

File: src/_pytest/pastebin.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def create_new_paste(contents):
     from urllib.request import urlopen
     from urllib.parse import urlencode
 
-    params = {"code": contents, "lexer": "python3", "expiry": "1week"}
+    params = {"code": contents, "lexer": "text", "expiry": "1week"}
```
Comment (nicoddemus) [inline src/_pytest/pastebin.py]: Hi @goerz, wondering, why is "python3" no longer recognized? I mean, is there another, or it doesn't really make any difference what the lexer is? 🤔

Comment (goerz) [reply src/_pytest/pastebin.py]: “python3” is still recognized, and in fact it works fine for uploading pytest output most of the time. It just fails for specific outputs, like the one I included in the report for #5806. I haven’t really looked into the internals of [pinnwand](https://github.com/supakeen/pinnwand) (the backend for bpaste.net), but I would assume that the specified lexer is used to parse the pasted text (for syntax highlighting?). Obviously, the pytest output is never valid Python code, and I’m really not sure why this particular output causes bpaste to return an http error, while other, very similar output goes through without an error. In any case, though, the “text” lexer is definitely the semantically correct choice, as pytest output isn’t something that can be parsed.

Comment (nicoddemus) [reply src/_pytest/pastebin.py]: Fair enought, thanks!

**Comment by goerz** [comment]
Thanks for merging! Should I follow up in some way on the "needs backport"?

**Comment by nicoddemus** [comment]
Oh sure, feel free to cherry pick on the `4.5-mainteance` branche. 👍

**Pull request merged**